components/Layout.vue

```diff
@@ -121,19 +110,21 @@ onMounted(() => {
     });
   }, observerOptions);
 
-  document
-    .querySelectorAll('section')
-    .forEach(contentSection => {
-      observer.observe(contentSection);
-    });
+  const sections = document.querySelectorAll('section');
+  if (sections.length) {
```

Comment:
Added to highlight first section by default. This wasn't happening previously with the overview at the top.